lib/livebook/hubs.ex

```diff
@@ -129,7 +129,7 @@ defmodule Livebook.Hubs do
 
   defp disconnect_hub(hub) do
     Task.Supervisor.start_child(Livebook.TaskSupervisor, fn ->
-      Process.sleep(30_000)
+      Process.sleep(5_500)
```

Comment:
```suggestion
      # Since other processes may have been communicating
      # with the hub, we don't want to terminate abruptly and
      # make them crash, so we give it some time to shut down.
      Process.sleep(5_500)
```

Reply:
Please check what happens if you have a notebook within the hub and the hub is deleted. :)

Reply:
It reports the issues to import the file, like the stamping and the hub id

![image](https://github.com/livebook-dev/livebook/assets/6402997/c49cfa77-10dd-44de-aa64-702b17d8ee7c)

Reply:
But, changing to 5s doesn't show the flash messages. Perhaps use a task to send the broadcast?